crates/acct-types/src/messages.rs

```diff
@@ -66,4 +66,9 @@ impl MsgPayload {
     pub fn data(&self) -> &[u8] {
         &self.data
     }
+
+    // todo: until ssz
+    pub fn as_ssz_bytes(&self) -> Vec<u8> {
```

Comment:
This is definitely not SSZ since we have to encode this in DA and we want to minimize bytes as much as possible, and there's no need to be able to merkle prove just a part of a message.

Reply:
It should use our codec system in the strata-common repo, which will be more amenable to zero-copy.